libs/ngx-videogular/core/src/lib/services/vg-fullscreen-api/vg-fullscreen-api.service.ts

```diff
@@ -174,4 +174,11 @@ export class VgFullscreenApiService {
       document[this.polyfill.exit]();
     }
   }
+
+  @HostListener('document:fullscreenerror', ['$event'])
+  onFullscreenerror(event: Event) {
+    console.log('An error occurred changing into fullscreen: ', event);
```

Comment:
Does the library in general log to console?

Reply:
@andreashermann no.
I added it for us.

Should i remove?
Should i build version from this pr?

Reply:
I don't think we should log additionally in the library as it cannot be controlled.
it's ok to build a version from this PR

Reply:
logs removed

Reply:
+ @49ing/ngx-videogular@5.0.5